issue#2077. Creating class for keeping all user's attributes

Nastsi opened the pull request.

Nastsi committed:
issue#2077. Creating class for keeping all user's attributes

Nastsi committed:
issue#2077. Creatin empty line in the end of class UsersAttributes

jacob-l commented on RedirectApplication/RedirectApplication/Controllers/ValuesController.cs:
```diff
@@ -17,20 +17,24 @@ public class ValuesController : ApiController
-            var MobileOrNot = HttpContext.Current.Request.Browser.IsMobileDevice.ToString(); //true - request was made by Mobile device
-            var userIP = HttpContext.Current.Request.UserHostAddress.ToString(); //Now it`s ::1 because it's running locally
-            var language = Request.Headers.AcceptLanguage.ToString().Substring(0, 2); //The most used language
+            UsersAttributes user = new UsersAttributes();
```
In the most places, we use var instead of declaration of Type.

jacob-l commented on RedirectApplication/RedirectApplication/Models/UsersAttributes.cs:
```diff
@@ -0,0 +1,19 @@
+
+namespace RedirectApplication.Models
+{
+    public class UsersAttributes
```
Inconsistent naming of properties. We use CamelCase with public properties in the class

jacob-l replied:
First letter of the properties must be capitalized
Example:
Url, Browser, MobileOrNot

jacob-l reviewed (commented):
Wrong commit messages.

I remember that guys send you link to artical with common rules for messages.

Description of commit should be something like "Create class ..." instead of creating.

Nastsi committed:
issue#2077 Using CamelCase and  var

1.  Var instead of declaration of Type.
2. Changing name of public fields in UsersAttributes

Nastsi committed:
issue#2077 Capitalize first letter of the properties

Nastsi committed:
issue#2077 Merge master
